python_examples/sim_speeds/basic_speed_plot.py

```diff
@@ -0,0 +1,40 @@
+# Code to plot runtime from .txt file
+
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def get_seconds(time_str):
```

Comment:
Even outside of epiabm itself, I think we have been trying to add complete docstrings to functions, especially ones like these that are very dependent on the form of the input.